qvet-web/src/components/UnresolvedCheckRun.tsx

```diff
@@ -0,0 +1,158 @@
+import { LoadingButton } from "@mui/lab";
+import { Alert, AlertTitle, Stack } from "@mui/material";
+import { Link } from "react-router-dom";
+
+import useBaseSha from "src/hooks/useBaseSha";
+import {
+  checkRunOverruleContext,
+  getCheckRunLevel,
+} from "src/hooks/useCheckRuns";
+import useCommitStatus from "src/hooks/useCommitStatus";
+import useSetCommitState from "src/hooks/useSetCommitState";
+import { CheckRun } from "src/octokitHelpers";
+import { CheckRunGlobalConfig } from "src/utils/config";
+
+import UserLink from "./UserLink";
+
+// Alert for a check run that is either incomplete or failed
+export default function UnresolvedCheckRun({
+  checkRun,
+  config,
+}: {
+  checkRun: CheckRun;
+  config: CheckRunGlobalConfig;
+}): React.ReactElement {
+  const baseSha = useBaseSha();
+  const level = getCheckRunLevel(checkRun, config);
+
+  switch (level) {
+    case "info":
+      return <InfoCheckRun checkRun={checkRun} />;
+    case "embargo":
+      if (baseSha.data) {
+        return <EmbargoCheckRun checkRun={checkRun} sha={baseSha.data} />;
+      }
+  }
+
+  return <></>;
```

Comment:
This is a little weird, but I guess you expect never to hit this case? Maybe worth a
```suggestion
  console.error(`Unknown embargo level ${level}`);
  return <></>;
```